pkg/gardenlet/controller/shoot/shoot/reconciler_reconcile.go

```diff
@@ -312,6 +312,18 @@ func (r *Reconciler) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			SkipIf:       o.Shoot.HibernationEnabled || skipReadiness,
 			Dependencies: flow.NewTaskIDs(deployKubeAPIServer),
 		})
+		scaleEtcdAfterRestore = g.Add(flow.Task{
+			Name:         "Scaling main and events etcd after restoration from backup",
+			Fn:           flow.TaskFn(botanist.ScaleUpETCD).RetryUntilTimeout(defaultInterval, helper.GetEtcdDeployTimeout(o.Shoot, defaultTimeout)),
+			SkipIf:       !botanist.IsRestorePhase() || o.Shoot.HibernationEnabled || skipReadiness,
+			Dependencies: flow.NewTaskIDs(waitUntilKubeAPIServerIsReady),
+		})
+		_ = g.Add(flow.Task{
+			Name:         "Waiting until main and events etcd scaled up after restoration from backup",
```

Comment:
We can directly deploy the etcd events with three replicas, as etcd-events do not require restoration from a backup. However, the drawback of deploying etcd events with three replicas is that it require three volume attachments. This increase in volume attachments could potentially raise the likelihood of encountering volume attachment issues, which in turn could lead to an increase in the time required for CPM.
So, I'm fine with it.

Reply:
just wording can be improved that etcd events is not restoring from backups.

Reply:
Yes exactly, that is why I decided to deploy it with only 1 replica as well. That also sped up the process quite a bit.